e2e-cypress/Jenkinsfile.template

```diff
@@ -11,7 +11,7 @@ odsComponentPipeline(
   podContainers: [
       containerTemplate(
         name: 'jnlp',
-        image: "${dockerRegistry}/ods/jenkins-agent-nodejs20:@agent_image_tag@",
+        image: "${dockerRegistry}/ods/jenkins-agent-nodejs22:@agent_image_tag@",
```

Comment:
we should also make this change in the Jenkinsfile use to provision the quickstarter -> https://github.com/roicarrera/ods-quickstarters/blob/master/e2e-cypress/Jenkinsfile